tests/integration/integration_test.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
nit. aiobravado is python3 only, this line is not needed as by default utf-8 is used

Reply:
Yes I know that :)
We have the same pre-commit rules as bravado, it's one of the few files we don't get merge conflicts...